web/packages/teleterm/src/ui/StatusBar/StatusBar.tsx

```diff
@@ -17,15 +17,27 @@
  */
 
 import { Fragment } from 'react';
+import { useTheme } from 'styled-components';
 
-import { Flex, Text } from 'design';
+import { ButtonPrimary, Flex, Text } from 'design';
+
+import { useAppContext } from 'teleterm/ui/appContextProvider';
+import { ConnectionStatusIndicator } from 'teleterm/ui/TopBar/Connections/ConnectionsFilterableList/ConnectionStatusIndicator';
 
 import { AccessRequestCheckoutButton } from './AccessRequestCheckoutButton';
 import { ShareFeedback } from './ShareFeedback';
 import { useActiveDocumentClusterBreadcrumbs } from './useActiveDocumentClusterBreadcrumbs';
 
-export function StatusBar() {
+export function StatusBar(props: { onAssumedRolesClick(): void }) {
   const breadcrumbs = useActiveDocumentClusterBreadcrumbs();
+  const ctx = useAppContext();
+  const theme = useTheme();
+  const rootClusterUri = ctx.workspacesService.getRootClusterUri();
+
+  const assumedRoles = Object.values(
+    ctx.clustersService.getAssumedRequests(rootClusterUri)
+  );
+  const assumedRolesText = assumedRoles.flatMap(r => r.roles).join(', ');
```

Comment:
Say I have three requests, one for role A, one for role B, and one for roles A and B. If I assume all three, the status bar is going to say "A, B, A, B".

I don't think it's something we have to address right now, it's probably best to let people use it and see what they say. But I guess in certain contexts it might cause some problems that we aren't able to uniquely identify an access request.

Reply:
Yeah, I wish we had a simpler/shorter way to identify an access request. 
There's not much space on the status bar, so I wanted to keep the text as concise as possible.

Another thing is that for resource access requests I show the role name in the status bar, but not in the selector. I didn't want to clutter the menu with too much information (but it's only visible on hover). In other words, there's no visible connection between the role in the status bar and the menu. But OTOH, Web UI works the same.